### DISCUSSION TITLE
visualize_preprocess not showing anything

### DISCUSSION THREAD

**Discussion by aledifil** [miniscope/minian#228] [Q&A]
Hi, I need a little help. I used to run Minian in the past without any issues.
I'm now running it again on some old data after I denoised the original videos, to have cleaner results.
Everything seems to work well (glow removal for example is ok) until the denoise process.
The denoise function alone is actually working well (I can see it changes varr_ref), but I cannot use the visualization_preprocess to check the best parameter.
The same happens with the background removal step: the function works but the visualize_preprocess does not show anything.
It also does not produce any errors.
![Screenshot 2022-11-22 153914](https://user-images.githubusercontent.com/158089/203358041-ec718a8d-5091-4fc3-a179-47baa4de7a16.png)
Any help is greatly appreciated!

**Comment by kirolle** [comment]
I have the same issue, running the demo pipeline in a Jupyter notebook. Are there any known workarounds?

**Comment by lgries-lt** [comment]
I am also running into this problem!
